pkg/status/templates/process-agent.tmpl

```diff
@@ -25,6 +25,7 @@ Process Agent
   {{- else}}
   System Probe Process Module Status: Not running
   {{- end}}
+  Process Language Detection Enabled: {{ if and .language_detection.enabled (eq .language_detection.enabled "true") }}True{{ else }}False{{ end }}
```

Comment:
Is this exported from the core agent?

Reply:
I think all the agents have a `status` command

Reply:
Also, if we use sprig we could simplify this to `.language_detection.enabled | title` ([docs](https://masterminds.github.io/sprig/strings.html#:~:text=above%20returns%20hello-,title,-Convert%20to%20title)). It's already in the go mod file, so it should be safe to use

Reply:
Apparently it does not work: `panic: template: /process-agent.tmpl:28: function "title" not defined`.

Also I was wrong. This variable is read from a `stats` object which is obtained by sending a request to the process-agent. 

Also, using directly ` {{ expvars.language_detection_enabled }` returns `true` or `false`, without a capital letter.

For testing, I deployed the agent with the environment variable `DD_LANGUAGE_DETECTION_ENABLED=true` in both the agent and process agent. Here are the results:
```
❯ k -n datadog-agent-helm exec datadog-agent-linux-dv8lt -- agent status | grep -i language
Defaulted container "agent" out of: agent, trace-agent, process-agent, system-probe, security-agent, init-volume (init), init-config (init), seccomp-setup (init)
2023-08-16 15:50:08 UTC | CORE | WARN | (pkg/util/log/log.go:664 in func1) | Unknown environment variable: DD_PROCESS_CONFIG_LANGUAGE_DETECTION_ENABLED
2023-08-16 15:50:08 UTC | CORE | WARN | (pkg/config/config.go:1578 in LoadCustom) | Unknown environment variable: DD_PROCESS_CONFIG_LANGUAGE_DETECTION_ENABLED
  Process Language Detection Enabled: True
❯ k -n datadog-agent-helm exec datadog-agent-linux-dv8lt -c process-agent -- agent status | grep -i language
2023-08-16 15:52:22 UTC | CORE | WARN | (pkg/util/log/log.go:664 in func1) | Unknown environment variable: DD_PROCESS_CONFIG_LANGUAGE_DETECTION_ENABLED
2023-08-16 15:52:22 UTC | CORE | WARN | (pkg/config/config.go:1578 in LoadCustom) | Unknown environment variable: DD_PROCESS_CONFIG_LANGUAGE_DETECTION_ENABLED
  Process Language Detection Enabled: True
```

Reply:
@AliDatadog I think you need to do
```go
template.Funcs(sprig.FuncMap())
)
```

Reply:
@just-chillin I think the `title` function provided by `sprig` works for string input https://github.com/Masterminds/sprig/blob/e4c0945838d570720d876a6ad9b4568ed32317b4/functions.go#L116

We have a `formatTitle` function but apparently it doesn't work with boolean input so for now I guess it's alright to leave it like this.